### DISCUSSION TITLE
Does the existing API support trimming video?

### DISCUSSION THREAD

**Discussion by hellostephanie2022** [rust-av/rust-av#182] [General]
Hi, I checked the source code and found that `TimeInfo` actually contains `pts` and `dts`. I tried to use the existing API to fulfill the trim video requirement, but obviously I failed. Can you help me with an example? Or it's not supported yet, is there anything I can do to help?

**Comment by lu-zero** [comment]
We are going to overhaul the API soon, trimming video, as in remuxing a video selecting only a part of it, is not yet supported in `ave`, but the information provided by the current demuxing API are enough to do it. Sorry for the very belated reply.
